### DISCUSSION TITLE
bdsocket.library / uaenet.device greyed out under Linux

### DISCUSSION THREAD

**Discussion by Juppstein** [BlitterStudio/amiberry#1158] [Q&A] [answered]
Hi, pretty new to Amiberry. I'm trying it under Ubuntu 22.04 LTS and wondered if I'm doing something wrong that bdsocket and uaenet are disabled. Do I need to place libraries manually? Also, JIT seems to be disabled as well. Sorry for those newbie questions but I wasn't able to find anything about it in the wiki or elsewhere.

**Comment by solskogen** [comment]
You cannot enable them when emulator is running.

**Comment by midwan** [comment, marked as answer]
bsdsocket.library can only be enabled/disabled when the emulation is not running.
JIT is only available in ARM for the moment, in case you're running this on x86_64 for example.
